packages/office-ui-fabric-react/src/components/TextField/TextField.Props.ts

```diff
@@ -88,16 +88,36 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
   description?: string;
 
   /**
-  * String for addon.
+  * String for prefix (deprecated)
   */
   addonString?: string;
 
   /**
-  * Custom render function for addon
+  * String for prefix
+  */
+  prefix?: string;
+
+  /**
+  * String for suffix
+  */
+  suffix?: string;
+
+  /**
+  * Custom render function for prefix (deprecated)
```

Comment:
use the `@deprecated` flag in the comments (search for this in other comments). This will make it show up in deprecations in documentation.

Reply:
👍